src/test/lib.c

```diff
@@ -8,11 +8,11 @@ int main(int argc, char ** argv){
 
   if (rank == 0){
     char * param[] = {"./ior", "-a", "DUMMY"};
-    IOR_test_t * res = ior_run(3, param, MPI_COMM_SELF, stdout);
```

Comment:
Instead of removing the returned value, shouldn't we check that the returned value is != NULL?

Reply:
same. Thanks for the review!